### ISSUE TITLE
Validation freq

### ISSUE THREAD

**Issue by RAMIhamza** [open-mmlab/OpenUnReID#46]
Hey,

When I run two trainings (strong baseline) with the same config file, except for val_freq which is the frequency of validation (2 and 5 in my case). I surprisingly get two different curves of the training cross entropy loss. 
I believe that even though the model is in eval mode, it still changes some of its settings. Am I wrong ?

Thanks in advance.

**Comment by RAMIhamza** [comment]
Please find bellow some specifications of my setup.
I'm working with the following versions:
- Python 3.6.9
- Torch '1.7.1+cu110'

Also the two plots of the cross entropy loss that correspond to the two trainings (strong baseline) with different val frequencies.

![image](https://user-images.githubusercontent.com/32485004/139837959-bcf13ef2-2333-4b9f-bf0c-1cc83b32c286.png)
Grey line : val_freq = 5
Orange line: val_freq = 2

**Comment by chongyangwang-song** [comment]
@RAMIhamza Hi， I study this code recently， could we have some Communication？

**Comment by RAMIhamza** [comment]
Yeah sure ! 
Here is my mailing address : hamza.rami.student@gmail.com
